cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -134,19 +134,35 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
       query("SHOW DATABASES", assertDatabasesShown) {
         resultTable()
       }
-      p("This command is optionally idempotent, with the default behavior to throw an exception if the database already exists. " +
-        "Appending `IF NOT EXISTS` to the command will ensure that no exception is thrown and nothing happens should the database already exist. " +
-        "Adding `OR REPLACE` to the command will result in any existing database being deleted and a new one created.")
-      query("CREATE DATABASE customers IF NOT EXISTS", ResultAssertions(r => {
-        assertStats(r, systemUpdates = 0)
-      })) {}
-      query("CREATE OR REPLACE DATABASE customers", ResultAssertions(r => {
-        assertStats(r, systemUpdates = 2)
-      })) {
-        p("This is equivalent to running `DROP DATABASE customers IF EXISTS` followed by `CREATE DATABASE customers`.")
+      section("Handling Existing Databases", "administration-databases-create-database-existing", "enterprise-edition") {
+        p("This command is optionally idempotent, with the default behavior to throw an exception if the database already exists. " +
+          "Appending `IF NOT EXISTS` to the command will ensure that no exception is thrown and nothing happens should the database already exist. " +
+          "Adding `OR REPLACE` to the command will result in any existing database being deleted and a new one created.")
+        query("CREATE DATABASE customers IF NOT EXISTS", ResultAssertions(r => {
+          assertStats(r, systemUpdates = 0)
+        })) {}
+        query("CREATE OR REPLACE DATABASE customers", ResultAssertions(r => {
+          assertStats(r, systemUpdates = 2)
+        })) {
+          p("This is equivalent to running `DROP DATABASE customers IF EXISTS` followed by `CREATE DATABASE customers`.")
+        }
+        note {
+          p("The `IF NOT EXISTS` and `OR REPLACE` parts of this command cannot be used together.")
+        }
       }
-      note {
-        p("The `IF NOT EXISTS` and `OR REPLACE` parts of this command cannot be used together.")
+      section("Options", "administration-databases-create-database-options", "enterprise-edition") {
```

Comment:
I do not know if there exists some sort of `clustering` flag similar to the `enterprise-edition` one. If it does it should probably be added here.

Reply:
There does not appear to be a flare for this, just 'enterprise-edition', 'aura', 'fabric' and 'deprecated'. We could discuss adding one with the docs team if you think it is important.

Reply:
I don't think it is super important if it does not exist already